src/pages/patientView/genomicOverview/tracksHelper.ts

```diff
@@ -77,39 +77,25 @@ function getChmEndsPerc(chms: Array<any>, total: any) {
     return ends;
 }
 
-/**
- * storing chromesome length info
- */
-export const genomeBuilds: Map<string, string> = new Map([
-    ['hg19', 'GRCh37'],
-    ['37', 'GRCh37'],
-    ['hg38', 'GRCh38'],
-    ['38', 'GRCh38'],
-    ['mm10', 'GRCm38'],
-]);
-
 export type ChromosomeSizes = {
     genomeBuild: string;
     chromosomeSize: number[];
 };
 
 const referenceGenomeSizes: {
     [genomeBuild: string]: number[];
-} = chromosomeSizes.reduce(
-    (map: { [genomeBuild: string]: number[] }, next: ChromosomeSizes) => {
-        map[next.genomeBuild] = next.chromosomeSize || [];
-        return map;
-    },
-    {}
-);
+} = _(chromosomeSizes)
+        .keyBy(entry => entry.genomeBuild)
+        .mapValues(entry => entry.chromosomeSize)
+        .value();
 
 export function getChmInfo(genomeBuild: string) {
```

Comment:
I would be in favor of refactoring this function name to `getGenomeCoordinates`  or something

Reply:
It is not a function, but a data structure (a map). The data that the keys are pointing to are number that each represent a chromosome size. So, I think the name is appropriate. Why would you like a name change?

Reply:
I was indicating `getChmInfo` ;)

Reply:
Ah ok, this function build a whole collection of functions that translate chromosomal location to positions on a 2d canvas for drawing the tracks. I agree that the name `getChmInfo` can be improved. I also think that `getGenomeCoordinates` does not cover the content correctly. What do you think of `getChromosomeCoordinateTranslation`?

Reply:
Why does it not cover the content? What the methods does it calculate the relative coordinates of the points, based on the chromosome length and the start/end position. What about `getRelativeCoordinates`? I am sure you can work from here. Please get this merged.

Reply:
Update to getRelativeCoordinates